bottle.py

```diff
@@ -1925,6 +1925,8 @@ def wrapper(*a, **ka):
                 rv = callback(*a, **ka)
             except HTTPError as error:
                 rv = error
```

Comment:
`HttpError` is a subclass of `HttpResponse` anyway, so you may as well remove this `except`